package.json

```diff
@@ -13,6 +13,7 @@
   },
   "dependencies": {
     "@quasar/extras": "^1.15.5",
+    "dayjs": "^1.11.7",
```

Comment:
I notice you installed `dayjs` but I see no use of this lib in the code.
Where exactly are you using this library?

Reply:
I was trying something, but i should remove it.